### DISCUSSION TITLE
Learning the syntax of Neorg

### DISCUSSION THREAD

**Discussion by refaelsh** [nvim-neorg/neorg#1235] [Q&A] [answered]
Where would I go for documentation on the syntax?
I've searched the entire README several times up and down and can't see the syntax documentation/explainer.
I only saw the video in the README, is there something in text form?

Thanks.

**Comment by sebtheiler** [comment, marked as answer]
In NeoVim, execute `:help Neorg`. The `.norg` syntax is the first thing covered. Vhyrro also has a [YouTube series](https://www.youtube.com/watch?v=NnmRVY22Lq8&list=PLx2ksyallYzVI8CN1JMXhEf62j2AijeDa) on Neorg that covers its basics, including some of the syntax.

**Reply by refaelsh** [reply]
I understand. Thank you very much for your answer.

**Comment by pysan3** [comment]
Checkout my tutorial for norg format!

https://github.com/pysan3/Norg-Tutorial

**Reply by refaelsh** [reply]
Cool. Thanks :-)

**Comment by max397574** [comment]
additional to the other two answers there is also the repo with the formal specification in this org
